modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -173,6 +185,9 @@ struct GAPI_EXPORTS_W_SIMPLE OpenVINO {
     */
     GAPI_WRAP
     OpenVINO& cfgCacheDir(const std::string &dir) {
+        if (is_initialized_from_map) {
+            throw std::runtime_error("ep::OpenVINO cannot be changed if created from the parameters map.");
```

Comment:
Could you please use `cv::util::throw_error` and `logic_error` instead?

https://github.com/opencv/opencv/blob/d804c0b6078b94d94296ac9f75af2e2e695b2336/modules/gapi/include/opencv2/gapi/infer/ov.hpp#L109-L112

Reply:
Using `cv::util::throw_error` now.

Reply:
Thanks